src/language/ast.ts

```diff
@@ -209,11 +209,10 @@ export const QueryDocumentKeys: {
   Field: ['alias', 'name', 'arguments', 'directives', 'selectionSet'],
   Argument: ['name', 'value'],
 
-  FragmentSpread: ['name', 'directives'],
+  FragmentSpread: ['name', 'arguments', 'directives'],
   InlineFragment: ['typeCondition', 'directives', 'selectionSet'],
   FragmentDefinition: [
     'name',
-    // Note: fragment variable definitions are deprecated and will removed in v17.0.0
```

Comment:
are we removing these? I don't recall why we decided to remove these cc @yaacovCR

Reply:
Afaik it was decided because the feature wasn't being developed on